doc/news/create_changes_h.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+## ---------------------------------------------------------------------
+##
+## Copyright (C) 2016 by the deal.II authors
+##
+## This file is part of the deal.II library.
+##
+## The deal.II library is free software; you can use it, redistribute
+## it, and/or modify it under the terms of the GNU Lesser General
+## Public License as published by the Free Software Foundation; either
+## version 2.1 of the License, or (at your option) any later version.
+## The full text of the license can be found in the file LICENSE at
+## the top level of the deal.II distribution.
+##
+## ---------------------------------------------------------------------
+
+#
+# This script creates changes.h from the contributions in the subfolders
+# of ./doc/news.
+#
+# The script needs to be executed as 
+#   ./create_changes_h.sh
+# from ./doc_news.
+
+
+
+if test ! -d incompatibilities -o ! -d general -o ! -d specific ; then
+  echo "*** This script must be run from ./doc/news!"
+  exit 1
+fi
+
+echo INCOMPATIBILITIES
+cat header_incompatibilities > changes.h
+echo "<ol>" >> changes.h	
+ARRAY=($(ls incompatibilities | sort -r))
```

Comment:
i am not a bash-guru, maybe document the logic and variables throughout the script so that it is more clear for others.